MetaMask/metamask-extension
# Ramps buy/sell button What's New

## **Description**
The Ramps team is introducing an off-ramps functionality within the Portfolio. This What's New was created to make users aware of this feature. Now the Buy button will be changed to a Buy & Sell button.
 
## **Manual testing steps**

1. Install a new MetaMask or update it to the latest build
2. This What's New item should appear on the second position after the Snaps announcement. If you've already seen the Snaps announcement, it'll be the first item you see.
3. CTA should click through to https://portfolio.metamask.io/sell/build-quote

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**
3 What's New items:
- Introducing new and refreshed Swaps!
- Looking for your account details or block explorer URL?
- Introducing MetaMask Snaps

### **After**

2 What's New items:
- Introducing MetaMask Snaps
- Ramps buy & sell button

![Screenshot 2023-09-19 at 6 10 02 PM](https://github.com/MetaMask/metamask-extension/assets/6951929/aef72807-1389-411b-955b-61a3b44d6391)


## **Related issues**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
